Scripts/Macros.cmd.txt

```diff
@@ -0,0 +1,7 @@
+b=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat $*

```

Comment:
Do you plan to do something similar for Mac development?

Reply:
Yes -- that way a developer on either platform has access to the same shortcuts, even if the implementation is different.